### DISCUSSION TITLE
Send one non-mutated message to check if definition is ok

### DISCUSSION THREAD

**Discussion by cyberfreaq** [jtpereyda/boofuzz#678] [Q&A] [answered]
Hi, I'm trying to figure out the easiest way to send 1 message to the target where nothing is mutated and only the default values are used so that I can check with Wireshark for example if I got the message definition right.

Right now I'm using index_end=1 in the Session and a simple type with a fuzz_values parameter set to a valid value, while having all the fuzzable parameters of all the other Blocks set to False. 

However, this seems to be a bit complicated for that purpose. Is there a simpler way?

**Comment by jtpereyda** [comment, marked as answer]
I believe you'll want [`Session.feature_check()`](https://boofuzz.readthedocs.io/en/latest/source/Session.html#boofuzz.Session.feature_check) instead of `Session.fuzz()`.

I'm not sure we have docs on the CLI, but if you're using it, there is a `--feature-check` flag on the `boo fuzz` command.

**Reply by cyberfreaq** [reply]
Exactly what I was looking for, thank you!
